migrations/versions/503d27d0a21a_update_dino_cube_and_fmc_events_.py

```diff
@@ -0,0 +1,29 @@
+"""Update Dino Cube and FMC events descriptions
```

Comment:
On the filename, can you prefix it with `044_` to indicate it's the 44th DB migration? The previous revision (555566667777) has `043` in front. Prefixing the migration scripts with numbers helps keep them in order in IDEs, etc. It's not a requirement of the migration platform, but it's something I like to follow because I think it helps.

Reply:
Oh, sure! I'm not sure why alembic didn't do this (as I think it should've done)